packages/cli-plugin-metro/src/commands/start/runServer.ts

```diff
@@ -101,6 +101,7 @@ async function runServer(_argv: Array<string>, ctx: Config, args: Args) {
     secure: args.https,
     secureCert: args.cert,
     secureKey: args.key,
+    // @ts-ignore
     hmrEnabled: true,
```

Comment:
We can just remove this - the types are correct, it's not an option of the runServer API any more: https://github.com/facebook/metro/commit/158ee58b0b33c421b37a9265a3b3f1552b32ee28

(It *is* still a no-op option on the Metro CLI.. we should clean that up on the Metro side in the next major)

Reply:
```suggestion
```

Reply:
Thanks, I'd hoped this was the case!